test/jdk/java/awt/Checkbox/CheckboxCheckerScalingTest.java

```diff
@@ -46,40 +46,47 @@
 public class CheckboxCheckerScalingTest {
     private static Frame frame;
     private static Checkbox checkbox;
-    private static BufferedImage imageAfterChecked;
+    private static volatile Point point;
     private static volatile boolean checkmarkFound = false;
```

Comment:
`checkmarkFound` need not to be `volatile`. Although declaring it as volatile won't add any overhead but it will prevent optimization. For this test, it won't have much impact though.